.gitignore

```diff
@@ -57,3 +57,6 @@ docs/_build/
 # PyBuilder
 target/
 
+# VSCode
+.vscode
+
```

Comment:
Nitpick: It looks like at some point, someone bullied me into including a whole load of crap in this file, giving rise to the temptation for each person to customize this project to suit their needs. In my opinion, it's an anti-pattern to keep a copy of the aggregate of all development environments in every project. I personally keep common ignores in a global ignores file and reference that from my git system ignores, such that it's unnecessary to supply these for each and every project I maintain.

But since this project has a grab-bag of ignores from #184, I can't fault you for including your preferences. In the future, this will get reverted, so I recommend creating a global .gitignore. Maybe I'll write a blog about it explaining a better way.

Reply:
I know that my ignorance is possibly irritating, but I have no idea what a "global .gitignore" is.  I will look that up and read about it.  I only updated the project's .gitignore to make sure I wasn't messing something up by using VSCode.  I meant no harm, and hope you can chalk it up to my ignorance (which I am trying to correct!!)

Reply:
In [this section](https://blog.jaraco.com/skeleton/#ignoring-artifacts), I've documented my strategy for `.gitignore`.

Reply:
Thank you for that specific reference.  I also found this [article](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) which I'm assuming (based on looking at `~/.gitconfig`) uses

```
git config --global core.excludesfile ~/.gitignore_global
```

to edit `.gitconfig` for the user.  Your approach of directly editing `.gitconfig` seems both easier and also helped me learn how `git` is globally configured.